From 6b69c02033cd6d65837dc01166e646119e357573 Mon Sep 17 00:00:00 2001
From: Narendhar Reddy <quic_nare@quicinc.com>
Date: Tue, 21 May 2024 11:05:59 +0530
Subject: [PATCH] net : remove config_cfg80211 check

Remove config_cfg80211 check as it causing compilation
issue in linux6.6.
in linux 6.6 struct wireless_dev *ieee80211_ptr is
introduced as part of #ifdef CFG80211 which
was causing issue in ath compilation as the config is
disabled.

Change-Id: I5496ace5d329776f645b09ede29bf0e921c7d440
Signed-off-by: Narendhar Reddy <quic_nare@quicinc.com>
---
 include/linux/netdevice.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/include/linux/netdevice.h b/include/linux/netdevice.h
index d28b08c47985..d485fcb5857f 100644
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -2250,9 +2250,7 @@ struct net_device {
 #if IS_ENABLED(CONFIG_AX25)
 	void			*ax25_ptr;
 #endif
-#if IS_ENABLED(CONFIG_CFG80211)
 	struct wireless_dev	*ieee80211_ptr;
-#endif
 #if IS_ENABLED(CONFIG_IEEE802154) || IS_ENABLED(CONFIG_6LOWPAN)
 	struct wpan_dev		*ieee802154_ptr;
 #endif
-- 
2.34.1

